### ISSUE TITLE
Not All Unlabeled Data are Equal: Learning to Weight Data in Semi-supervised Learning 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#55]
# TL;DR
In semi-supervised learning, the weights of unlabeled data are usually treated uniformly, but they propose a method for automatically determining the weights of individual data, which can be incorporated into existing loss systems such as FixMatch and significantly improve the accuracy.
![image](https://user-images.githubusercontent.com/24524018/91296981-7c256880-e7d8-11ea-8618-5097afcb3276.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.01293

# Submission Dates(yyyy/mm/dd)
2020/07/02 

# Authors and institutions
Zhongzheng Ren, Raymond A. Yeh, Alexander G. Schwing
- University of Illinois

# Methods
After several updates using semi-supervised losses, use Hessian to update the individual weights of the data.
![image](https://user-images.githubusercontent.com/24524018/91296981-7c256880-e7d8-11ea-8618-5097afcb3276.png)
![image](https://user-images.githubusercontent.com/24524018/91296992-80518600-e7d8-11ea-8385-b85a776be09f.png)

# Results
Improved on CIFAR10, SVHN
![image](https://user-images.githubusercontent.com/24524018/91297001-834c7680-e7d8-11ea-8b62-a45b6973a205.png)

# Comments
